R/WorkbookClass.R

```diff
@@ -3135,10 +3138,17 @@ Workbook$methods(
       drawings_rels[[sheet]],
       sprintf(
         '<Relationship Id="rId%s" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/image" Target="../media/image%s.%s"/>',
-        imageNo,
+        imageRelNo,
```

Comment:
Do you know if this was always just approximately right?

Reply:
Apologies if I have misunderstood your question, but my understanding of `openxlsx` is that because it only created a `drawings_rels[[sheet]]` element when it created a `drawings[[sheet]]` element and it only created one `drawings_rels[[sheet]]` element at that time, the index of a `drawings[[sheet]]` element (`imageNo`) *always* matched the index of the associated `drawings_rels[[sheet]]` element.

Since this would no longer be true with the addition of a `drawings_rels[[sheet]]` element for hyperlinks, it is only now necessary to reference the indexes of the two lists independently.

Reply:
Thanks, makes sense. I just wasn't sure anymore.